lib/ui/amount.js

```diff
@@ -119,7 +119,7 @@ class Amount {
         return this.toBits(num);
       case `m${pkg.unit}`:
         return this.toMilli(num);
-      case pkg.currency:
+      case pkg.unit:
```

Comment:
This could maybe accept both, they are synonymous ones just an abbreviation:

```
case pkg.unit:
case pkg.currency:
  return this.toCoins(num)
```